samples/camera/README.rst

```diff
@@ -0,0 +1,30 @@
+.. camera:
+
+Camera Sample Application
+###################################
+
+Overview
+********
+This project demos camera samples.
+It should work with any platform featuring a I2C peripheral interface and a CMOS sensor interface.
+It does not work on QEMU.
+If there is display device on board, the demo displays camera stream on the display device.
+
+Requirements
+************
+This project requires a CMOS sensor connected to board.
+
+Building and Running
+********************
+This sample can be built for a ``mimxrt1050_evk`` board.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/camera
+   :board: mimxrt1050_evk
+   :goals: build flash
+
+To run this sample, a CMOS sensor should be present connected to the board.
+The sample outputs the frames per second number on the console in capture
+mode and preview mode periodically.
+If there is LCD pannel present on the board, the sample outputs the camera
```

Comment:
You call it a "display device" on line 11, but an "LCD panel" here.  You should pick one term and use it consistently.

Reply:
Fixed.